AFTests/ApiV2/ApiV2Tests.cs

```diff
@@ -0,0 +1,84 @@
+﻿using ApiV2Data.DTOs;
+using ApiV2Data.Fixtures;
+using FluentAssertions;
+using RestSharp;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using Xunit;
+using XUnitTestCommon.Utils;
+
+namespace AFTests.ApiV2
+{
+    [Trait("Category", "FullRegression")]
+    [Trait("Category", "ApiV2Service")]
+    public class ApiV2Tests : IClassFixture<ApiV2TestDataFixture>
+    {
+        private ApiV2TestDataFixture fixture;
+        private Dictionary<string, string> emptyDict = new Dictionary<string, string>();
+
+        public ApiV2Tests(ApiV2TestDataFixture fixture)
+        {
+            this.fixture = fixture;
+        }
+
+        [Fact]
+        [Trait("Category", "Smoke")]
+        [Trait("Category", "Pledges")]
+        public async void PledgeEndPoint()
+        {
+            string url = fixture.ApiEndpointNames["Pledges"];
+
+            //create pledge
```

Comment:
Get all pledges, get single pledge, create/edit/delete pledge should be separate test cases instead of one. At the moment if any of the above mentioned fails, the whole test case will fail and the rest will not be tested.